backends/text-to-speech/windows/windows-text-to-speech.cpp

```diff
@@ -29,6 +29,15 @@
 #include <basetyps.h>
 #include <windows.h>
 #include <Servprov.h>
+
+// Mingw-w64 is missing symbols for two guids declared in sapi.h which are used
+//  by sphelper-scummvm.h. Mingw32 doesn't include any sapi headers or libraries.
+#if defined(__MINGW32__)
```

Comment:
```suggestion
#if defined(__MINGW32__) && defined(__MINGW64_VERSION_MAJOR)
```
This should allow the TTS to still be compiled with the old mingw (if somebody goes through the effort of using MS SAPI files with mingw).